### PR TITLE
WEB-88: fetch only staff users (and many other code improvements)

### PR DESCRIPTION
**Pull request by dqd**
- only staff users are included
- faster dependency fetching
- new `load_dev_db` flags
- code cleanup
- tests improved

### CODE REVIEW

File: .github/workflows/code_check.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    - name: Set up Python 3.x
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
+    - name: Install dependencies
+      run: |
+        python -m pip install pyflakes
+    - name: Lint with pyflakes
+      run: |
+        pyflakes .
```
Comment (knyghty) [inline .github/workflows/code_check.yml]: Can you configure your editor (and perhaps add `.editorconfig` to this project) to add newlines at EOF?

File: dev_db/utils.py [hunk 2]
```diff
@@ -37,9 +25,10 @@ def model_name(m):
 
 def get_all_fields(instance):
     # follow forward relation fields
-    normal_fields = instance.__class__._meta.fields
-    many_to_many_fields = instance.__class__._meta.many_to_many
-    private_fields = instance.__class__._meta.private_fields
+    model = instance.__class__ if hasattr(instance.__class__, "_meta") else instance
```
Comment (knyghty) [inline dev_db/utils.py]: Why do we sometimes have the class, and sometimes have the instance?

File: dev_db/utils.py [hunk 3]
```diff
@@ -5,29 +5,17 @@
...
-        if instances:
-            max_id = getattr(instances[0], "id", 0)
-    return max_id
+    last = m._default_manager.order_by("-pk").first()
+    return last.pk if last and isinstance(last, int) else 0
 
 
 def get_field_names(m):
     field_names = [f.name for f in m._meta.fields]
     return field_names
```
Comment (knyghty) [inline dev_db/utils.py]: Maybe we should change `m` -> `model`

Comment (knyghty) [inline dev_db/utils.py]: We could simplify a bit. I think it's just as clear:

```suggestion
    return [f.name for f in m._meta.fields]
```

File: dev_db/utils.py [hunk 4]
```diff
@@ -5,29 +5,17 @@
 
 
 def get_max_id(m):
```
Comment (knyghty) [inline dev_db/utils.py]: What is this used for and how does it interact with e.g. uuid primary keys?

File: dev_db/management/commands/load_dev_db.py [hunk 5]
```diff
@@ -1,66 +1,84 @@
...
         parser.add_argument(
-            "--limit",
-            default=None,
-            dest="limit",
-            type=int,
-            help="Allows you to limit the number of tables, used for testing purposes only",
-        )
-        parser.add_argument(
-            "--skipcache",
+            "--yes",
```
Comment (knyghty) [inline dev_db/management/commands/load_dev_db.py:33]: It would be good to add `-y`

File: dev_db/management/commands/load_dev_db.py [hunk 6]
```diff
@@ -1,66 +1,84 @@
...
-            dest="format",
-            help="Specifies the output serialization format for fixtures.",
-        )
-        parser.add_argument(
-            "--indent",
-            default=4,
-            dest="indent",
-            type=int,
-            help="Specifies the indent level to use when pretty-printing output",
+            "--input",
```
Comment (knyghty) [inline dev_db/management/commands/load_dev_db.py:25]: Maybe this makes more sense as a positional argument, but I'm not sure.

File: .github/workflows/code_check.yml (hunk 1 above)
Comment (dqd) [reply .github/workflows/code_check.yml]: No problem, I will do that. However, the default Stream's settings is `insert_final_newline = false` (see https://github.com/GetStream/stream/blob/master/.editorconfig#L10).

**Review by knyghty — APPROVED** [review]
Ok, let's get this merged, test it a bit more and hopefully get a pypi release out.

**Pull request merged**